### PR TITLE
tickets/DM-13264: Add query for job by metric value

### PR DESCRIPTION
**Pull request by SimonKrughoff**
@timj have a look at this additional use case.  I can write a new requirement if you think it needs one.

### CODE REVIEW

File: LDM-592.tex [hunk 1]
```diff
@@ -1126,6 +1126,14 @@ \section{Science Quality and Reliability Engineering}
 
 }
 
+\usecase{SQR17}{Retrieve Job data based on metric values.}{%
+
+\item
+The continuous integration system publishes \texttt{Job} data including \texttt{Metric}s, \texttt{Measurement}s, and data associated with the \texttt{Measurement}s for display within SQuaSH.
+At some later point, the SQuaRE scientist retrieves all \texttt{Job}s that were run in the last month where the measurement of the photometric repeatablity is greater than 15 mas.
```
Comment (timj) [inline LDM-592.tex]: This is a distinct part of the use case so should have its own `\item`. Also, photometric repeatability is mentioned here but is not mentioned in the first part. Be more explicit in the first half.

Comment (SimonKrughoff) [reply LDM-592.tex]: OK, I'll try again.

**Comment by SimonKrughoff** [comment]
@timj have a look at this now.  I hope it's better.

File: LDM-592.tex [hunk 2]
```diff
@@ -1126,6 +1126,18 @@ \section{Science Quality and Reliability Engineering}
 
 }
 
+\usecase{SQR17}{Retrieve Job data based on metric values.}{%
+
+\item
+On a regulart cadence, the continuous integration system runs a script that generates measurements of various properties of a test dataset: e.g. photometric and astrometric repeatability.
```
Comment (timj) [inline LDM-592.tex]: "regulart".

**Review by timj — APPROVED** [review]
Looks fine to me if @TallJimbo is okay with it. I assume I associate it with DMS-MWBT-REQ-0088?

File: LDM-592.tex (hunk 2 above)
Comment (SimonKrughoff) [reply LDM-592.tex]: Good catch.  Fixed.

**Comment by TallJimbo** [comment]
I'm fine with this.  There are several ways I can imagine implementing this, and while I'm not sure all of them will work in detail, I'm pretty confident that we can find one that will.

As an aside, I think the name `Job` is way too overloaded to mean something metric-specific.  Our nomenclature around different units of processing has yet to solidify (Gen. 3 currently has "Runs" that produce "Collections" full of "Quanta", and I believe these are all subsets of what @MichelleGower calls a "Campaign", and "Batch Jobs" fit in there somewhere), but when it does we should think about which of those is most naturally associated with a bundle of metrics and derive a new name from that.  I'm not sure it's worthwhile trying to get ahead of the game by using more generic language here, but I thought it might be something for you to consider.

**Comment by SimonKrughoff** [comment]
I agree with your comment regarding the name Job.  We are saved somewhat here because it is an object in the lsst namespace, so we can be more specific by saying `verify.Job`.

**Pull request merged**